app/allocation/controllers/person-assign/confirmation.js

```diff
@@ -0,0 +1,9 @@
+// const MoveCreateConfirmationController = require('../../../move/controllers/create/confirmation')
+
+const PersonAssignBase = require('./base')
+
+class ConfirmationController extends PersonAssignBase {}
```

Comment:
You can drop controllers that inherit from the base if you include the base controller in the form wizard config.

Reply:
Ah. They need to inherit from the base controller of assignment for cancel URL and setting move.

Feels like there should be a nicer way to make that more flexible between the wizards...

Reply:
Yep. I did start simply extending the create classes, but the person search results just didn't work. Will look at it again to see what my (probably obvious) mistake was